Server/db/MongoDB.js

```diff
@@ -195,7 +195,9 @@ const getAllMonitors = async (req, res) => {
 const getMonitorById = async (req, res) => {
   try {
     const monitor = await Monitor.findById(req.params.monitorId);
-    return monitor;
+    const checks = await Check.find({ monitorId: monitor._id });
+    const monitorWithChecks = { ...monitor.toObject(), checks };
+    return monitorWithChecks;
   } catch (error) {
     throw error;
```

Comment:
Are we going to be sending a response to the client for the error or?

Reply:
Errors are thrown to the controller, and then they are thrown and caught by the server middleware.

So this error will ultimately be sent to the client!